### ISSUE TITLE
"Unable to load user angeltypes." in all-angels View

### ISSUE THREAD

**Issue by bastinat0r** [engelsystem/engelsystem#189]
I updated the englsystem for eh15 and run into the following problem: If I go to Admin->All Angels I only get the message "Unable to load user angeltypes.". Everything else seems to work fine, eg if I go to Arrived Angels there is no error.
I also tried to delete and recreate the db, but it does not work yet :(
Do you have any hints what to look for when debugging this error? I'm currently grep-ing through the code, but I'm by no means fluent in php ;-)

**Comment by bastinat0r** [comment]
So I tried to digg a little deeper.
I made debug outputs for $user and $unconfirmed_user_angeltypes in user_angeltypes_controller.php because greping the errormessage led me there.
The function that throws the error does so because User_unconfirmed_AngelTypes($user) === false what is the case according to my debug output.
The fnord is: I don't understand where this value comes from.
I used the excact SQL-Query from User_unconfirmed_AngelTypes which evaluates to an empty set which shouldn't throw the error. If I skip the check the page works.
